app/src/main/java/com/sdp/movemeet/backend/BackendManager.java

```diff
@@ -32,7 +32,7 @@
      * @param field the field to which the array corresponds in the backend.
      * @return a Task, the success of which determines the success of the operation.
      */
-    Task<?> update(String path, String field, String value);
+    Task<?> update(String path, String field, String value, String method);
```

Comment:
What does the method ?

Reply:
The `update` function of the Firebase Firestore abstraction takes as argument this `method` argument to specify if we want to remove (with `method` being set as "remove") or add (with `method` being set to another String) an element from an array stored in a document of Firebase Firestore.